app/src/org/commcare/activities/CommCareActivity.java

```diff
@@ -376,14 +376,7 @@ public void stopTaskTransition() {
         }
     }
 
-    /**
-     * Display exception details as a pop-up to the user.
-     *
-     * @param e Exception to handle
-     */
-    public void displayException(Exception e) {
-        String title =  Localization.get("notification.case.predicate.title");
-        String message = Localization.get("notification.case.predicate.action", new String[]{e.getMessage()});
+    public void displayException(String title, String message) {

```

Comment:
This method was previously showing a very specific type of error, even though it had the generic name `displayException`. The new method below, `displayCaseListFilterException`, now has the behavior that `displayException` had previously, and this method is now appropriately generic.